apps/web/src/services/routes.ts

```diff
@@ -18,6 +18,24 @@ export enum BackofficeRoutes {
   rewards = 'rewards',
 }
 
+const PUBLIC_ROOT_PATHS = {
+  setup: '/setup',
+  login: '/login',
+  magicLinks: '/magic-links',
+  invitations: '/invitations',
+}
```

Comment:
I prefer this instead of using the regex in middleware. I think we have more control. I left the middleware regex for nextjs urls stuff

Reply:
agree